web/templates/user_group_settings/cannot_deactivate_group_banner.hbs

```diff
@@ -1,8 +1,22 @@
 <div class="cannot-deactivate-group-banner main-view-banner error">
     <p class="banner-text">
-        {{t "To deactivate this group, you must first remove all permissions assigned to it."}}
+        {{#if group_used_for_permissions}}
+            {{t "To deactivate this group, you must first remove all permissions assigned to it."}}
+        {{else}}
+            {{#tr}}
+                To deactivate this group, you must first remove it from all other groups. This group is currently a subgroup of: <z-supergroup-names></z-supergroup-names>.
+                {{#*inline "z-supergroup-names"}}
+                    {{#each supergroups}}
+                        <a class="view-group-members" data-group-id="{{group_id}}" href="{{settings_url}}">{{group_name}}</a>
+                        {{~#unless @last}}, {{/unless~}}
```

Comment:
Tried using `util.format_array_as_list_with_conjunction` for this, but since we want period at the end of the sentence was not able to figure out a correct way to do that with translation.

Reply:
OK, probably not important enough to spend more time doing that.